pandas/io/sql.py

```diff
@@ -1645,10 +1645,10 @@ def check_case_sensitive(
         if not name.isdigit() and not name.islower():
             # check for potentially case sensitivity issues (GH7815)
             # Only check when name is not a number and name is not lower case
-            from sqlalchemy import inspect
+            sqlalchemy = import_optional_dependency("sqlalchemy", errors="ignore")
```

Comment:
Is there a need to use `import_optional_dependency` here? (I would assume that if you get at this point without having sqlalchemy installed, you already got an error about that earlier? eg the init of this class also imports from sqlalchemy)

Reply:
indeed, no need, might as well just import with an alias